docs/staticwebapp.config.json

```diff
@@ -1,5 +1,11 @@
 {
-	"routes": [],
+	"routes": [
+		{
+			"route": "/docs/fcdocstest",
+			"redirect": "/docs/api/v1",
```

Comment:
Note: this doesn't exist yet. If we're merging this now:
```suggestion
			"redirect": "/docs/apis",
```

Reply:
Would be good to test `/apis`!